### DISCUSSION TITLE
Give up after some amount of update backlog

### DISCUSSION THREAD

**Discussion by NotAFile** [piqueserver/piqueserver#701] [Feature Requests]
with @Jipok's new update loop (#575), the server will try to catch up on physics ticks to guarantee one second of physics happens per second. However, it may happen that for some reason, the server is too slow to run a second of simulation per second. In that case, it will get stuck endlessly. It would be nice to have an option to bail out of this cleanly.

**Comment by Jipok** [comment]
https://github.com/piqueserver/piqueserver/pull/628

**Comment by NotAFile** [comment]
Issues shouildn't be closed until the relevant commits have been merged
